packages/ui/src/components/Header/Header.tsx

```diff
@@ -42,6 +42,7 @@ const Header = ({ handleDrawerOpen }: Props) => {
                 <NavLink
                   key={name}
                   to={path}
+                  data-cy={`button-navigate-${name.toLowerCase().replace(/ /g, '-')}`}
```

Comment:
Ok with this? We're making the links programmatically so think I have to do the same for the test tags and doing this means we can have consistent looking locators eg

`editNamesMenuOption: () => cy.get('[data-cy=menu-option-edit-names]')`

instead of 

  `editNamesMenuOption: () => cy.get('[data-cy=menu-option-Edit names]')`

Reply:
Sure, sounds good.